libzkbob-rs-wasm/src/params.rs

```diff
@@ -24,15 +27,30 @@ impl From<Params> for Parameters<Engine> {
 #[wasm_bindgen]
 impl Params {
     #[wasm_bindgen(js_name = "fromBinary")]
-    pub fn from_binary(input: &[u8]) -> Result<Params, JsValue> {
-        Self::from_binary_ext(input, true, true)
+    pub fn from_binary(input: &[u8], precompute: bool) -> Result<Params, JsValue> {
```

Comment:
may be leave that method as it is, since we already have extended version for fine tuning? this way we won't need any changes on caller side

Reply:
Both `fromBinary` and `fromBinaryExtended` functions are used in the client library, so we need to make adjustments to the library regardless. However, it definitely makes sense. Fixed in https://github.com/zkBob/libzkbob-rs/pull/60/commits/19d72645069072c690f94e988be9b64c34e6ef3a.